src/test/java/jalau/at18/katas/multiples/MainTest.java

```diff
@@ -0,0 +1,14 @@
+package jalau.at18.katas.multiples;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import jalau.at18.katas.multiples.rodrigov.Main;
```

Comment:
Instead of importing the class, you can actually define this `MainTest` class to be in the same package `jalau.at18.katas.multiples.rodrigov`. You can do this by updating line 1

Reply:
i solved all the paths of my unit tests